src/main/java/core/basesyntax/service/ReadingCsvFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReadingCsvFile {
```

Comment:
The interface name 'ReadingCsvFile' could be more descriptive and follow Java naming conventions. Consider renaming it to 'CsvFileReader' to clearly reflect its purpose.